src/pages/YearPickerPage.js

```diff
@@ -68,10 +68,18 @@ class YearPickerPage extends React.Component {
         const routes = lodashGet(this.props.navigation.getState(), 'routes', []);
         const dateOfBirthRoute = _.find(routes, (route) => route.name === 'Settings_PersonalDetails_DateOfBirth');
 
-        if (dateOfBirthRoute) {
-            Navigation.setParams({year: selectedYear.toString()}, lodashGet(dateOfBirthRoute, 'key', ''));
+        const activeRoute = Navigation.getActiveRoute();
```

Comment:
Just noting that I think this is ok for now, but we need to fix the DatePicker component so it's reusable without hardcoding routes.

Reply:
@luacmartins can we create an issue for this? I don't mind doing it if you don't have the time today, just let me know

Reply:
created an issue for it here - https://github.com/Expensify/App/issues/20698